src/runtime/auto-prompt-manager-test.js

```diff
@@ -1540,36 +1589,42 @@ describes.realWin('AutoPromptManager', {}, (env) => {
       onCancel();
       await tick(2);
     }
-
-    function setupPreviousImpressionAndDismissals(
-      storedImpressions,
-      storedDismissals,
-      dismissedPrompts,
-      dismissedPromptGetCallCount
-    ) {
-      storageMock
-        .expects('get')
-        .withExactArgs(STORAGE_KEY_IMPRESSIONS, /* useLocalStorage */ true)
-        .returns(Promise.resolve(storedImpressions))
-        .once();
-      storageMock
-        .expects('get')
-        .withExactArgs(STORAGE_KEY_DISMISSALS, /* useLocalStorage */ true)
-        .returns(Promise.resolve(storedDismissals))
-        .once();
-      storageMock
-        .expects('get')
-        .withExactArgs(
-          STORAGE_KEY_DISMISSED_PROMPTS,
-          /* useLocalStorage */ true
-        )
-        .resolves(dismissedPrompts)
-        .exactly(dismissedPromptGetCallCount);
+  });
+  function setupPreviousImpressionAndDismissals(
```

Comment:
Could this `setupPreviousImpressionAndDismissals` method accept a single object param to improve readability instead of so many params? I'm finding the invocations of the method a little hard to read

Reply:
For example, instead of this...
```
setupPreviousImpressionAndDismissals(
        storageMock,
        null,
        null,
        null,
        1,
        null,
        true
      );
```

Perhaps this...
```
setupPreviousImpressionAndDismissals({
        storageMock,
        dismissedPromptGetCallCount: 1,
        getUserToken: true,
      });
```

Reply:
Actually, maybe someone could tackle this in a future PR